### ISSUE TITLE
stat_statements queryid exists sonce 9.4 version

### ISSUE THREAD

**Issue by pmpetit** [cybertec-postgresql/pgwatch2#80]
Hello,
on RDS 9.3 psql version, pg_stat_statements is 1.1, so the column queryid (in  queryid::text as tag_queryid, (...)
does not exists.

So i think version should be  9.4
![image](https://user-images.githubusercontent.com/4059185/41726906-69fa74a4-7573-11e8-8085-270903b696ae.png)

**Comment by kmoppel** [comment]
Good catch! Seems most people have upgraded to higher PG versions already:) But actually instead of just raising the version bar I'm thinking maybe would make sense to generate this "queryid" from the query text in the data fatching query. This would enable to use the pgwatch2 "top statements" and "single query details" dashboards still, together with URL linking from the former to the latter. Will take a look

**Comment by pmpetit** [comment]
Great !
thx a lot
